src/doc/guide.md

```diff
@@ -383,30 +388,32 @@ fn main() {
 }
 ```
 
-Putting `fn main() {` in each example is a bit tedious, so we'll leave that out
-in the future. If you're following along, make sure to edit your `main()`
-function, rather than leaving it off. Otherwise, you'll get an error.
+**Note**: Wrapping `fn main() { }` around each example is tedious, so we'll act
+like it's implied and stop explicitly writing it. As you follow along, make sure
+to edit your `main()` function rather than leaving it off. Otherwise, you'll get
+an error.
 
-In many languages, this is called a 'variable.' But Rust's variable bindings
-have a few tricks up their sleeves. Rust has a very powerful feature called
-'pattern matching' that we'll get into detail with later, but the left
-hand side of a `let` expression is a full pattern, not just a variable name.
-This means we can do things like:
+Here we have a `let` binding which declares a variable `x` and initializes it to
+the value 5. This is simple enough, but Rust's `let` bindings have a few tricks
+up their sleeves. Rust has a powerful feature called **pattern matching** that
+we'll discuss in detail later. The left hand side of a `let` binding is a
+**pattern**, not just a variable name. This means we can do things like:
 
 ```{rust}
 let (x, y) = (1i, 2i);
 ```
 
-After this expression is evaluated, `x` will be one, and `y` will be two.
-Patterns are really powerful, but this is about all we can do with them so far.
-So let's just keep this in the back of our minds as we go forward.
+After this statement is evaluated, `x` will be 1, and `y` will be 2. Patterns
```

Comment:
I'd like to leave it, unless you insist. We're talking about the numbers, sure, but we use the numerals in the program source, so it seems more reasonable to use the numerals in the prose as well.